packages/mds-db/geographies.ts

```diff
@@ -143,8 +163,12 @@ export async function deleteGeography(geography_id: UUID) {
   return geography_id
 }
 
-export async function publishGeography(params: PublishGeographiesParams) {
-  const { geography_id, publish_date } = params
+export async function publishGeography(params: PublishGeographiesParams): Promise<Geography> {
+  /* publish_date is a param instead of a default, because when a Policy is published,
```

Comment:
I wouldn't say we're using a param _instead_ of a default, considering we assign it a default value if none is passed.